pkg/store/redis/store.go

```diff
@@ -104,7 +166,7 @@ func (rs *redisStore) UpdateReports(scanJobID string, reports job.ScanReports) e
 }
 
 func (rs *redisStore) getKeyForScanJob(scanJobID string) string {
-	return fmt.Sprintf("%s:scan-job:%s", rs.namespace, scanJobID)
+	return fmt.Sprintf("%s:scan-job:%s", rs.cfg.Namespace, scanJobID)
 }
```

Comment:
Just for contrast with my other comments - I think it IS valuable having another function here, because the function name clarifies that this magic-looking string is the key; also, you're unlikely to need to know what the key is, precisely, when looking at the calling code

Reply:
Yup, I got your point!